appveyor/requirements.txt

```diff
@@ -3,7 +3,8 @@ pandas
 td-client
 pyspark
 pyarrow
-nose
+pytest
 mock
```

Comment:
We can delete `mock` from AppVeyor requirements as well.

Reply:
good catch. removed eb13381